OG format governace

```diff
@@ -17,20 +17,20 @@ The format will need to grow and evolve with time while being managed effectivel
 
 1. The team will analyze the issue formally. If the issue is not detailed enough or if there is no example associated with the issue, the issue will be rejected. If the issue is accepted for review, then the team will study it and provide an answer. The issue can lead to a modification of the format or not. If there is no consensus on the solution, the suggested solution will be rejected and the format will remain unchanged. However, a new branch can be created to demonstrate the benefit of the solution. The same issue can be submitted again with a demo.
 
-2. If a format update is required, then a pull will be made from the repository and updates made to the format description and supporting tools such as format checkers. The draft changes will go to OGDMTT members within the GitHub repository for appraval. Stakeholers will be noftifed of the change at this point.
+2. If a format update is required, then a pull will be made from the repository and updates made to the format description and supporting tools such as format checkers. The draft changes will go to OGDMTT members within the GitHub repository for approval. Stakeholers will be noftifed of the change at this point.
 
 ## Vocabaularies
-The vocabualry collections are more dynamic than the format. When terms are added to vocabualry colelctions there will be no need to update the format witht he vocabaulary contected being governed by a dedicate GitHub page for the collection.
+The vocabualry collections are more dynamic than the format. When terms are added to vocabualry collections there will be no need to update the format with the vocabulary contected being governed by a dedicate GitHub page for the collection.
 
-When new vocabularies are required or existing fields in teh format need to be constrained against a vocabualry then a simialr process as descibed for issues will be followed. There are additional options available for vocabualries:
-1. Identify an existing voabculary that already meets the needs of the OceanGliders community
+When new vocabularies are required or existing fields in the format need to be constrained against a vocabulary then a simialr process as descibed for issues will be followed. There are additional options available for vocabualries:
+1. Identify an existing vocabulary that already meets the needs of the OceanGliders community
 2. Create a scheme on an existing vocabulary where terms are too diverse to meet the needs of the OceanGliders community
```

Comment:
Shall we describe what a scheme is ?

Reply:
Link to the definition here?
https://www.bodc.ac.uk/resources/products/web_services/vocab/#uris

Reply:
That link is useful. Once it's added I think this PR is good to merge